src/client/components/app-core/buttons.css

```diff
@@ -38,10 +39,9 @@
 
 .button:hover,
 .button:focus {
+  background-color: var(--focus-color);
```

Comment:
the `border-color` should also change to the `--focus-color`.

Reply:
Added focus and hover styles for `.primary-button`.